mage_ai/frontend/components/CodeBlock/CommandButtons/index.tsx

```diff
@@ -175,6 +193,44 @@ function CommandButtons({
           </>
         )}
 
+        {BlockTypeEnum.SCRATCHPAD === block.type && (
+          <Spacing mt={PADDING_UNITS}>
+            <FlyoutMenuWrapper
+              compact
+              items={convertBlockMenuItems}
+              left={-UNIT * 22}
```

Comment:
Did the `UNIT * 22` amount come from somewhere? Or was it trial and error to get a length that positioned the menu well?

Reply:
Trial and error. Should I make it a constant, or try to find some other way to standardize?

Reply:
should be fine then